### PR TITLE
Qwik: fix all failed tests

### PR DESCRIPTION
**Pull request by hdJerry**

### CODE REVIEW

File: qwik-graphql-tailwind/src/components/gists/gists.spec.tsx [hunk 1]
```diff
@@ -1,29 +1,24 @@
 import { createDOM } from '@builder.io/qwik/testing';
 import { describe, vi, it } from 'vitest';
 import { Gists } from '~/components/gists';
-import { GistItem } from './types';
+// import { GistItem } from './types';
```
Comment (Megio) [inline qwik-graphql-tailwind/src/components/gists/gists.spec.tsx:4]: Probably we can remove this comment

File: qwik-graphql-tailwind/src/components/gists/gists.spec.tsx [hunk 2]
```diff
@@ -1,29 +1,24 @@
...
-  },
-];
+// const MOCK_GISTS: GistItem[] = [
+//   {
+//     id: '123',
+//     description: '123',
+//     name: 'gist-test',
+//     url: 'https://url-gists.com',
+//   },
+// ];
```
Comment (Megio) [inline qwik-graphql-tailwind/src/components/gists/gists.spec.tsx:13]: Same here

File: qwik-graphql-tailwind/src/components/gists/gists.spec.tsx (hunk 1 above)
Comment (hdJerry) [reply qwik-graphql-tailwind/src/components/gists/gists.spec.tsx:4]: Thanks

**Review by Megio — APPROVED** [review]
🚀

**Pull request merged**